WebSite/Redirect.aspx.cs

```diff
@@ -39,6 +39,21 @@ protected void Page_Load(object sender, EventArgs e)
                         Response.Headers.Add("Location", value);
                         Response.StatusCode = (int)HttpStatusCode.Redirect;
                         break;
+                    }
+                case "HardcodedResponseRedirectToGoogle":
```

Comment:
These three cases are not exposed in the UI in any way. I'm just checking that that's intentional; that you are, for example, driving them from a test framework, perhaps?

Reply:
Yeah, that's the case here. The three new cases will be used by the test framework only. "HardcodedResponseRedirectToGoogle" is used in negative tests (where redirect should always occur in that case)

So, not planning to update the UI for these new cases. Having said that, they could still be used for any demo purposes if really required, as the respective urls can still be typed into the browser address bar. However, I think the existing cases exposed in the UI are sufficient for demo purposes and not adding new links in the UI will keep the Redirect page somwhat 'simpler'.

Reply:
Perfect. Merging in.